dataflux_core/tests/fake_gcs.py

```diff
@@ -57,8 +57,12 @@ def list_blobs(
         return results
 
     def blob(self, name: str):
```

Comment:
It might make more sense to just create a defaulted kwarg (e.g. exists=true), that you can pass in to specify when a bucket should exist. That way we don't overload the name parameter with additional functionality.

Reply:
KW arg addition shouldn't break any existing tests

Reply:
Blob method is called from the actual code like this https://github.com/GoogleCloudPlatform/dataflux-pytorch/blob/main/dataflux_pytorch/lightning/dataflux_lightning_checkpoint.py#L88 

So there is no way for me to explicitly pass False when needed. Let me know if I mis-understood your comment

Reply:
Gotcha, since this fake_gcs code is only accessed during testing can you just manually assign the flag as "not-existing" in the tests? (e.g. my_blob.exists=False) to get the right behavior rather than naming it? If that doesn't work it can go ahead with naming.

Reply:
I dont get access to my_blob (again since it is called only from the prod code) but I have changed this to default arg with missing_path = False.